src/Components/DataRow.jsx

```diff
@@ -76,7 +76,7 @@ class DataRow extends Component {
         }
 
         return (
-            <td>
+            <td className="action-cell">
```

Comment:
Should this be `rddt` as well?

```suggestion
            <td className="rddt-action-cell">
```